packages/ckeditor5-utils/src/env.js

```diff
@@ -71,7 +73,15 @@ const env = {
 		 *
 		 * @type {Boolean}
 		 */
-		isRegExpUnicodePropertySupported: isRegExpUnicodePropertySupported()
+		isRegExpUnicodePropertySupported: isRegExpUnicodePropertySupported(),
+
+		/**
+		 * Indicates that the environment supports at least [Input Events Level 1](https://www.w3.org/TR/input-events-1/)
+		 * (`input` and `beforeinput` events).
+		 *
+		 * @type {Boolean}
+		 */
+		areInputEventsLevel1Supported: areInputEventsLevel1Supported( global.window )
```

Comment:
IMHO we should stick to code style for the boolean properties so `isInputEventsLevel1Supported` and that seems reasonable because we want to check here if the Level 1 of the Input Events is supported.